demo/src/index.js

```diff
@@ -19,6 +19,10 @@ class Demo extends React.Component {
     };
   }
 
+  static onEventMethod = (name, ...args) => {
```

Comment:
oooo. what are the implications of using a `static` here? i dont think i've seen us use this here before

Reply:
We did this for the `class-methods-use-this` line rule. Since we don't reference `this` within the method—the rule asks us to make the method static—which makes sense in this context as it's more of a helper method.